app/process_proposal.go

```diff
@@ -122,7 +122,11 @@ func (app *App) ProcessProposal(req abci.RequestProcessProposal) (resp abci.Resp
 		return reject()
 	}
 
-	dah := da.NewDataAvailabilityHeader(eds)
+	dah, err := da.NewDataAvailabilityHeader(eds)
```

Comment:
[Question] I'm curious to know if a malicious player can cause a validator to panic by using corrupted data. If that is indeed possible, would triggering a panic be considered a reasonable response?

Reply:
I don't understand the second part of the question, do you mind rephrasing?

I don't think it's possible for a malicious proposer to cause a validator to panic. If it were possible, then a malicious proposer could halt the chain by causing all honest validators to `panic`. I don't think `panic` is a reasonable response for an honest validator if they encounter a malicious block. In this scenario, an honest validator should reject the block and remain online.

Reply:
> If that is indeed possible, would triggering a panic be considered a reasonable response?

Thanks for the clarification. 🙏 
My question about panic does not apply to `process_proposal` as unlike `prepare_proposal` it does not panic (confused the two).